transform/mattermost-analytics/models/intermediate/product/configs/int__config_ldap.sql

```diff
@@ -0,0 +1,101 @@
+{{
+    config({
+        "materialized": "table",
+        "snowflake_warehouse": "transform_xs"
+    })
+}}
+
+select event_id
```

Comment:
An alternative would be to use [dbt_utils.union_relations](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#union_relations-source). It would preserve columns and make the code a bit more readable.